modules/live-migration-metrics-information.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="live-migration-metrics-information"]
+= Information about live migration metrics
+
+The following table shows you the available metrics and the label values populated from the `openshift_network_operator_live_migration_procedure` expression. Use this information to monitor progress or to troubleshoot the migration.
+
+
+.Live migration metrics
+[cols="1a,1a",options="header"]
+|===
+| Metric | Label values
+| 
+*`openshift_network_operator_live_migration_blocked:`*::
++
+--
+A Prometheus gauge vector metric. A metric which contains a constant `1` value labeled with the reason that the CNI live migration might not have began. This metric is available when the CNI live migration has started by annotating the `Network` custom resource. +
```

Comment:
```suggestion
A Prometheus gauge vector metric. A metric which contains a constant `1` value labeled with the reason that the CNI live migration might not have started. This metric is available when the CNI live migration has started by annotating the `Network` custom resource. +
```

Reply:
I'm a bit confused. The metric with a 1 indicates the migration hasn't started; but the metric only available when the _migration has started_ and only when _the live migration is blocked_.

Reply:
Weird but devs say it's accurate. I just confirmed once again.